proxy/src/main/java/com/velocitypowered/proxy/VelocityServer.java

```diff
@@ -332,18 +332,41 @@ private void loadPlugins() {
 
     try {
       Path pluginPath = Path.of("plugins");
+      Path updatePath = pluginPath.resolve("to-update");
+      Path outdatedPluginsPath = pluginPath.resolve("outdated");
 
-      if (!pluginPath.toFile().exists()) {
+      if (Files.notExists(pluginPath)) {
         Files.createDirectory(pluginPath);
-      } else {
-        if (!pluginPath.toFile().isDirectory()) {
-          logger.warn("Plugin location {} is not a directory, continuing without loading plugins",
-              pluginPath);
-          return;
-        }
+      }
+      if (Files.notExists(updatePath)) {
+        Files.createDirectory(updatePath);
+      }
+      if (Files.notExists(outdatedPluginsPath)) {
+        Files.createDirectory(outdatedPluginsPath);
+      }
+      if (!Files.isDirectory(pluginPath)) {
+        logger.warn("Plugin location {} is not a directory, continuing without loading plugins",
+                pluginPath);
+        return;
+      }
```

Comment:
I would move this check just after the plugins folder creation, to avoid creating secondary plugins folders if the main folder is not available

Reply:
Done